package.json

```diff
@@ -46,7 +46,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --watchAll=false",
+    "test": "npx cypress run",
```

Comment:
Can we create a script entry specifically for Cypress? I'm afraid that changing the default test script might break code climate or some other automations